resources/views/admin/routes/create.blade.php

```diff
@@ -0,0 +1,100 @@
+@extends('admin.layouts.master')
+@section('content')
+<div class="row">
+  <div class="col-md-12">
+    <div class="box box-primary">
+      <div class="box-header with-border">
+        <h3 class="box-title">{{ trans('admin_routes.edit_route') }}</h3>
+      </div>
+      <form role="form" action="{!!URL::route('admin.routes.store')!!}" method="POST">
+      {{csrf_field()}}
+        <div class="box-body">
+          <div class="form-group col-md-6">
+            <label>{{ trans('admin_routes.routes') }}</label>
+            <input type="texxt" class="form-control"  name="name">
+             <div class="form-group has-error"> 
+              <span class="help-block">{{$errors->first('name')}}</span>
+              </div>
+          </div>
+          <div class="form-group col-md-6">
+            <label>{{trans('admin_routes.distance')}} </label>
+            <input type="text" class="form-control"  name="distance">
+            <div class="form-group has-error"> 
```

Comment:
`has-error`, this class only show when the form input has error. But you always show it.